packages/analytics-js-integrations/package.json

```diff
@@ -101,7 +100,9 @@
     "lodash.pickby": "4.6.0",
     "lodash.tostring": "4.1.4"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "@rudderstack/analytics-js-common": "*"
+  },
```

Comment:
This change is not required. It's only needed for published packages.

Reply:
Addressed